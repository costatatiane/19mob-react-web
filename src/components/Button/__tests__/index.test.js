import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Button from '../index';

describe('Button', () => {
    // it('should be render button with label Test', () => {
    //     const element = renderer.create(<Button label="Test" />); 
    //     expect(element.toJSON()).toMatchSnapshot();
    // });

    // it('should be className equal btn', () => {
    //     const element = renderer.create(<Button label="Test" />); 
    //     expect(element).toHaveAttribute('class', 'btn');
    // });
});