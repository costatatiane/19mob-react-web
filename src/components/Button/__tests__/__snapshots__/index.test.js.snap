// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should be render button with label Test 1`] = `
<button
  className="btn"
>
  Test
</button>
`;
